src/lib/orbit-db-http-server/routes/get.js

```diff
@@ -0,0 +1,47 @@
+const get = async (req, res) => {
+  try {
+    // Get the database address from the request
+    const address = req.params[0]
+
+    // Get params on how we should output the results
+    const shouldStream = req.query.live || false
```

Comment:
A more concise way to do it
```suggestion
    const shouldStream = Boolean(req.query.live)
```